### ISSUE TITLE
New Wallet Impossible to use on MacOS and other questions

### ISSUE THREAD

**Issue by FaguiCurtain** [ethereum/mist#1073]
#### System information

```
Version: `Mist 0.8.1'
OS & Version: Mac OS 10.11.5 El Capitan
Node type: `eth/geth(default)`
```

I have reinstalled MIST  i had an old version; to do so, i put the icon of the older version in the trash bin and did
rm -r Ethereum

it took a while, but i managed to sync to the current block.
In the meanwhile i still had the same issue than with the previous version, with some files being quarantined by my AV software (Avast) that i had to manually restore.

Then , i go to the wallet. 

1) I think it created a new account which is fine for me, as i left no balance at all on the old wallet

but i don't remember one detail: in MIST there are 2 passwords, one password is for what ? purpose, and the other to send transactions. I think i was asked only to create 1 password when reinstalling. Why is it so ?

2) The wallet is like "frozen". When i click on one button, it takes ages (= several minutes) to go to the next screen, or it may not even come at all
for example when i click on the button "comptes" account up there, it took 5minutes to go to the next screen. and in the next screen, i can't input anything, and clicking back on comptes doesn't have any effect at all...

and after 30mns the window all went blank

<img width="1102" alt="capture d ecran 2016-08-05 a 20 05 17" src="https://cloud.githubusercontent.com/assets/11316512/17435093/75511b32-5b49-11e6-9701-fdcf8a454dc2.png">
<img width="1104" alt="capture d ecran 2016-08-05 a 20 15 49" src="https://cloud.githubusercontent.com/assets/11316512/17435092/75510976-5b49-11e6-9efb-79addfac96bc.png">

<img width="326" alt="capture d ecran 2016-08-05 a 21 36 34" src="https://cloud.githubusercontent.com/assets/11316512/17436803/c6ff2536-5b54-11e6-901f-b892a2e25490.png">

Node log file
...055399 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 790ccb7ccd98b77f7a227e7db34c8864a36c6968a1b028e98849ba6998cbc2d3
E0805 21:43:47.834567 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 79094aa05b52a4f4da48be749df6e0270d1e51111ce00284d12526f6a329fd13
E0805 21:43:48.233830 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 79094aa05b52a4f4da48be749df6e0270d1e51111ce00284d12526f6a329fd13
E0805 21:43:48.505859 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 790ccb7ccd98b77f7a227e7db34c8864a36c6968a1b028e98849ba6998cbc2d3
E0805 21:43:48.904610 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 790a29fa4110d04b20e3e1b63fcbda537ea386bfd34a8332a31f9f0c22d3c643
E0805 21:43:50.612217 trie/secure_trie.go:74] Unhandled trie error: Missing trie node 790a29fa4110d04b20e3e1b63fcbda537ea386bfd34a8332a31f9f0c22d3c643
I0805 21:44:26.336132 eth/handler.go:295] Peer 3f52484c661c2726 [eth/62]: timed out DAO fork-check, dropping

**Comment by luclu** [comment]
Please add an exception to `~/Library/Ethereum/chaindata` in Avast. (https://github.com/ethereum/mist/issues/573).

**Comment by FaguiCurtain** [comment]
i cannot add this to Avast

I don't know why but the Ethereum folder doesn't appear in the list of subdirectories of /Library
<img width="1040" alt="capture d ecran 2016-08-06 a 00 44 27" src="https://cloud.githubusercontent.com/assets/11316512/17442089/10461e6a-5b6f-11e6-9ce4-36c26ea0c9f0.png">

**Comment by luclu** [comment]
Try this: 
```
chflags nohidden ~/Library
```

**Comment by FaguiCurtain** [comment]
doesn't change from the pic above. the Ethereum folder is in the Library folder, but Avast cannot see it

**Comment by luclu** [comment]
Are you able to access the folder using finder after running that command?

**Comment by FaguiCurtain** [comment]
no

**Comment by luclu** [comment]
Try this: http://www.wikihow.com/Unhide-the-User-Library-Folder-on-a-Mac

**Comment by aureolopes** [comment]
Hi,

I'm having a problem to acess the wallet. 
Maybe someone can help me? 
Here is the message that i received:

"Node type: geth
Network: main
Platform: darwin (Architecure x64)

.../database.go:169] closed db:/Users/aureomarcus/Library/Ethereum/chaindata
I0806 18:49:02.144233 cmd/utils/flags.go:592] WARNING: No etherbase set and no accounts found as default
I0806 18:49:02.146665 ethdb/database.go:82] Alloted 512MB cache and 1024 file handles to /Users/aureomarcus/Library/Ethereum/chaindata
I0806 18:49:02.315023 ethdb/database.go:82] Alloted 16MB cache and 16 file handles to /Users/aureomarcus/Library/Ethereum/dapp
I0806 18:49:02.325187 eth/backend.go:172] Protocol Versions: [63 62], Network Id: 1
I0806 18:49:02.325410 eth/backend.go:201] Blockchain DB Version: 3
I0806 18:49:02.328691 core/blockchain.go:206] Last header: #0 [d4e56740…] TD=17179869184
I0806 18:49:02.328734 core/blockchain.go:207] Last block: #0 [d4e56740…] TD=17179869184
I0806 18:49:02.328759 core/blockchain.go:208] Fast block: #0 [d4e56740…] TD=17179869184
I0806 18:49:02.337477 p2p/server.go:313] Starting Server
Fatal: Error starting protocol stack: listen udp :30303: bind: address already in use"

**Comment by aureolopes** [comment]
I think i will open a new topic.. tks.

**Comment by aureolopes** [comment]
But i'm not sure about that. Maybe i will duplicate a topic unnecessarily. What i must do?

**Comment by luclu** [comment]
@aureolopes try to reboot your mac, create a new issue if this does not help.

**Comment by aureolopes** [comment]
Thank you, Luca!

**Comment by FaguiCurtain** [comment]
@luclu 
when i am in Finder i do see the Ethereum folder.
when i am on AVAST and i try selecting the Ethereum folder (as an exception) I don't see it in the Library folder.

**Comment by FaguiCurtain** [comment]
@luclu 
so i have managed to solve the problem for AVAST, 
http://superuser.com/questions/957463/avast-on-osx-how-to-add-hidden-files-and-folders-as-an-exception

However that doesn't solve my MIST problem, i cannot do anything

If i click on my main account icon or anything else, nothing happens

If i click on quitting MIST by clicking on the red button of the window, nothing happens as well

Here is my logfile

..mported 1 block(s) (0 queued 0 ignored) including 0 txs in 10.438213ms. #2038294 [3a086244 / 3a086244]
I0809 14:16:32.803436 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 16 txs in 83.620388ms. #2038295 [c0f60973 / c0f60973]
I0809 14:16:50.271002 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 2 txs in 8.062266ms. #2038296 [aff0d0c4 / aff0d0c4]
I0809 14:17:01.992873 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 4 txs in 7.779472ms. #2038297 [86436668 / 86436668]
I0809 14:17:21.092325 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 0 txs in 26.305225ms. #2038298 [a48c422a / a48c422a]
I0809 14:17:27.637427 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 3 txs in 5.587026ms. #2038299 [1692be29 / 1692be29]
I0809 14:17:28.599089 core/blockchain.go:963] imported 1 block(s) (0 queued 0 ignored) including 1 txs in 6.950332ms. #2038299 [a7e87875 / a7e87875]

**Comment by luclu** [comment]
Try to backup the `~/Library/Application\ Support/Mist` folder and restart Mist.

**Comment by FaguiCurtain** [comment]
@luclu 
what is inside this folder for info ?

**Comment by luclu** [comment]
That folder contains the database of your local (watched) custom contracts and custom token. As there have been issues with the event logging of certain contracts, this might help with a freezing Wallet interface.

**Comment by evertonfraga** [comment]
Probably fixed in 0.8.x series.
